manifests/rbac.authorization.k8s.io.yaml.in

```diff
@@ -0,0 +1,84 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
Is there any sort of ordering during deployment time that requires this manifest to be posted before any of the others that depend on the kubevirt-infra account?

Reply:
Worth investigating.